src/extension/options-page/DashboardComponents/ProfileBox.tsx

```diff
@@ -72,15 +71,22 @@ export default function ProfileBox({ persona, border }: Props) {
     const color = useColorProvider()
     const profiles = persona ? [...persona.linkedProfiles] : []
 
-    const providers = [...definedSocialNetworkWorkers].map(i => {
-        const profile = profiles.find(([key, value]) => key.network === i.networkIdentifier)
-        return {
-            network: i.networkIdentifier,
-            connected: !!profile,
-            userId: profile?.[0].userId,
-            identifier: profile?.[0],
-        }
-    })
+    const providers = [...definedSocialNetworkWorkers]
+        .filter(i => {
+            if (webpackEnv.genericTarget === 'facebookApp') {
```

Comment:
Can be simplied as `webpackEnv.genericTarget !== 'facebookApp'`

Reply:
maybe current is more clear